docs/grid_exchange_formats/cgmes/export.md

```diff
@@ -1,10 +1,51 @@
 # Export
 
-<span style="color: red">TODO</span>
+There are two main use-cases supported:
+ * Export IGM (Individual Grid Model) instance files. There is a single network and a unique CGMES modelling authority. 
+ * Export CGM (Common Grid Model) instance files. A network composed of multiple subnetworks, where each subnetwork is an IGM. 
 
-Please note that PowSyBl always exports CGMES networks as CGMES Node/Breaker networks without considering the topology 
+To select one of the use-cases, use the parameter **iidm.export.cgmes.cgm_export**.  
+
+In both cases, the metadata model information in the exported files is built from metadata information read from the input files and stored in IIDM or received through parameters. 
+Information received through parameters takes precedence over information available from original metadata models.
+
+## IGM (Individual Grid Model) export
+
+The user can specify the profiles to be exported using the parameter **iidm.export.cgmes.profiles**. The list of currently supported export instance files are: EQ, SSH, SV, TP. 
+
+If the IIDM network has at least one voltage level with node/breaker topology level, and the SSH or SV is requested in the export, and the TP is not requested, an error will be logged, as there could be missing references in the SSH, SV files to Topological Nodes calculated automatically by IIDM that are not present in the output.
+
+If the dependencies have to be updated automatically (see parameter **iidm.export.cgmes.update-dependencies** below), the exported instance files will contain metadata models where:
+* TP and SSH depend on EQ.
+* SV depends on TP and SSH.
+* EQ depends on EQ_BD (if present). EQ_BD is the profile for the boundary equipment definitions.
+* SV depends on TP_BD (if present). TP_BD is the profile for the boundary topology. Only for CGMES 2.4.
+
+The output filenames will follow the pattern `<baseName>_<profile>.xml`. The basename is determined from the parameters, or the basename of the export data source or the main network name. 
+
+Please note that when exporting equipment, PowSyBl always use the CGMES node/breaker level of detail, without considering the topology
 level of the PowSyBl network.
 
+## CGM (Common Grid Model) export
+
+When exporting a CGM, we need an IIDM network (CGM) that contains multiple subnetworks (one for each IGM). 
```

Comment:
I disagree with this part: if we want to export a CGM with EQ and TP files, we can do it too. We have 2 methods to export CGM, a quick one that you are describing and a more complex one manipulating extensions and exporting profile by profile. Both must be explained and illustrated here.

Reply:
Added documentation for the two ways of exporting CGM: quick and manual. Included code to illustrate both ways of generating the required files. Also added a unit test for the "manual" option, showing how metadata can be defined by the user by setting information in the related extension.